garak/resources/beast/__init__.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Implementation of BEAST attack from Sadasivan et al. "Fast Adversarial Attacks on Language Models In One GPU Minute"
+(https://arxiv.org/abs/2402.15570v1)
+
+Code derived from paper, with help from the official implementation:
+https://github.com/vinusankars/BEAST?tab=readme-ov-file
+
+Considerable inspiration was drawn from Dreadnode's implementation:
+https://github.com/dreadnode/research/blob/main/notebooks/Mistral%20-%20BEAST%20Beam%20Attack.ipynb
+"""
+
+from garak.resources.beast.beast_attack import run_beast
```

Comment:
Is this ever consumed? What is the intent in adding this as part of `__init__.py`?

Reply:
I pull it in as part of suffix.py -- it's the only public method in the module. Probably not necessary?